### PR TITLE
[plugin.video.jwtv-unofficial] v1.8.3

### PR DESCRIPTION
**Pull request by ca0abinary**
### Description
Fixed bug in subtitle setting

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.jwtv-unofficial/addon.py [hunk 1]
```diff
@@ -22,162 +22,158 @@
...
-		li = xbmcgui.ListItem(s.get('name'))
-		if 'rph' in s['images']:
-			li.setIconImage(s['images']['rph'].get('md'))
-			li.setThumbnailImage(s['images']['rph'].get('md'))
-		if 'pnr' in s['images']:
-			li.setProperty('fanart_image', s['images']['pnr'].get('md'))
-		xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, isFolder=(isStreaming == False))
+    isStreaming = mode[0] == 'Streaming'
+    for s in subcat_ary:
+        url = build_url({'mode': s.get('key')})
```
Comment (enen92) [inline plugin.video.jwtv-unofficial/addon.py]: Hum probably better to initialize the dictionary and just add the keys if they exist. The way you're doing this right now basically replaces the dictionary

File: plugin.video.jwtv-unofficial/addon.xml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   </extension>
 
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Access to JW Broadcasting</summary>
-    <description lang="en">This plugin allows easy access to the content found on tv.jw.org</description>
-    <disclaimer lang="en">This is not an offical plugin, please do not contact jw.org for support.</disclaimer>
+    <summary>Access to JW Broadcasting</summary>
+    <description>This plugin allows easy access to the content found on tv.jw.org</description>
+    <disclaimer>This is not an offical plugin, please do not contact jw.org for support.</disclaimer>
+    <license>Apache License, Version 2.0</license>
```
Comment (enen92) [inline plugin.video.jwtv-unofficial/addon.xml]: Just for your information, you can use the news tag in the addon.xml to provide a simple way of sharing what hás changed in adition to the changelog

**Comment by enen92** [comment]
Please rename your PR to match the default style: [script] version

**Comment by ca0abinary** [comment]
Done, thanks for the reminder.

**Pull request merged**